penciledgedev/Cimee-Mobile
# user login #171256092

### What does this PR do?
- it can authenticate users to use the mobile app

### How should you test this PR?
- Checkout to this branch
- clone the repo
- Make sure you have expo installed
- `run expo install` to install all dependencies
- ` run expo android` to run application on an AVD manager

### What are the relevant pivotal Tracker stories?
[#171256092](https://www.pivotaltracker.com/story/show/171256092)
